changelogs/fragments/1182-migrate-module-utils-data-set.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - module_utils/copy.py - migrate code to use ZOAU v1.3.0.
```

Comment:
The title of the pull request is for utils/data_set, but the content of this file says it is for utils/copy

Reply:
I might also categorize this one as `trivial` since this change doesn't affect the end-user experience.